Bug: 4.3.0 で yarn run build:prod が失敗する

Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        |`Debian 10.10`|
|GROWI     |`4.3.0`|
|node.js   |`v12.22.1`|
|npm       |`6.14.12`|
|Using Docker|no|
|Using [growi-docker-compose][growi-docker-compose]|no|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

How to reproduce? (再現手順)
---------------------------

1. run `yarn install`
1. run `yarn run build:prod`

What happens? (症状)
---------------------
fails to run `run build:prod`

```
growi@n19:~/growi$ yarn prestart
yarn run v1.22.5
$ npm run build:prod
npm ERR! missing script: build:prod
npm ERR!
npm ERR! Did you mean one of these?
npm ERR!     build:poll
npm ERR!     build:dev
npm ERR!     build:slack

npm ERR! A complete log of this run can be found in:
npm ERR!     /srv/growi/.npm/_logs/2021-07-15T04_10_32_247Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

What is the expected result? (期待される動作)
-------------------------------------------
run `yarn run build:prod` successfully.

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
     "build:dev:watch": "npm run build:dev:app:watch",
     "build:dev:watch:poll": "npm run build:dev:app:watch:poll",
     "build:dev": "yarn build:dev:app",
+    "build:prod": "yarn lerna run build --scope @growi/app --scope @growi/slack",
     "build:slack": "lerna run build --scope @growi/slack",
     "build": "npm run build:dev:watch",
     "build:poll": "npm run build:dev:watch:poll",

